window_view.cpp

```diff
@@ -1458,6 +1458,18 @@ void WindowView::keyboardAction(const SDL_Keysym& key) {
       inputQueue.push(UserInput(getDirActionId(key), Vec2(-1, -1)));
       mapGui->onMouseGone();
       break;
+  #ifndef RELEASE
+    case SDL::SDLK_TAB:
```

Comment:
Please move the whole case statement above so we don't duplicate the #ifndef RELEASE. Also { should be moved to the line above.